dev/benchmarks/test_apps/stocks/lib/stock_row.dart

```diff
@@ -77,8 +79,6 @@ class StockRow extends StatelessWidget {
                     ),
                   ),
                 ],
-                crossAxisAlignment: CrossAxisAlignment.baseline,
```

Comment:
I think this is an improvement. otherwise it is hard to tell what these properties were associated with.